apps/wallet-mobile/src/features/Initialization/DarkThemeAnnouncement/DarkThemeAnnouncement.tsx

```diff
@@ -30,12 +28,10 @@ export const DarkThemeAnnouncement = () => {
     return () => clearTimeout(timeout)
   }, [])
 
-  const {walletImplementationIdChanged} = useSetupWallet()
   const {resetToWalletSetupInit} = useWalletNavigation()
 
   const navigate = () => {
     setScreenShown()
-    walletImplementationIdChanged(HASKELL_SHELLEY.WALLET_IMPLEMENTATION_ID)
```

Comment:
missing `walletImplementationChanged('cardano-cip1852')`?

Reply:
There is something strange because this screen and `ChooseBiometricLoginScreen.tsx` go to the same place. I think this change has been lost in some merge, I'm going to review it

Reply:
ty @banklesss